source/Server/Ldap/GroupParentFinder.cs

```diff
@@ -13,20 +13,14 @@ public interface IGroupParentFinder
 
     public class GroupParentFinder : IGroupParentFinder
     {
-        readonly ILdapConfigurationStore store;
```

Comment:
We dont want/need to inject a `ConfigurationStore` directly, we should access settings from the `LdapContext`